Release notes for Istio 1.13.9 (#12053)
istio/istio.io: content/en/news/releases/1.13.x/announcing-1.13.9/index.md

---
title: Announcing Istio 1.13.9
linktitle: 1.13.9
subtitle: Patch Release
description: Istio 1.13.9 patch release.
publishdate: 2022-10-07
release: 1.13.9
---

> This release contains a patch for [CVE-2022-41715](https://github.com/golang/go/issues/55949) and bug fixes to improve robustness. This release note describes what’s different between Istio 1.13.8 and Istio 1.13.9.

Reviewer: Do we want to link to the actual CVE since we are calling it out?

Author: Do you have a recommended link to use here?

Reviewer: I don't. Maybe the PSWG does.

Author: @jacob-delgado Do you have a recommended link to use here?

{{< relnote >}}

## Changes

> - **Fixed** an issue where user can not delete `iop` resource with revision if `istiod` is not running.  ([Issue #40796](https://github.com/istio/istio/issues/40796))

Reviewer: ```suggestion
- **Fixed**  an issue where user can not delete the Istio Operator resource with revision if istiod is not running.  ([Issue #40796](https://github.com/istio/istio/issues/40796))
```

Author: Done.

- **Fixed** a bug where the return dynamically generated by `jwks` was not base64 encoded, causing Envoy to fail to parse it.

- **Fixed** an issue where a root namespace `Sidecar` configuration would be ignored.

> ## Security update

Reviewer: I would think that this section should go above the changes.

Author: Done.

> - Patch for [CVE-2022-41715](https://github.com/golang/go/issues/55949). Replaces all uses of `stdlib` `regexp` with the `golang` 1.19.2 `stdlib` implementation. This will guard against DOS via malformed regular expressions.

Reviewer: Maybe change golang to Go.

---
title: Announcing Istio 1.13.9
linktitle: 1.13.9
subtitle: Patch Release
description: Istio 1.13.9 patch release.

> publishdate: 2022-10-07

Reviewer: Will need to update this to the correct date (10-11)?

release: 1.13.9
---

> This release contains a patch for [CVE-2022-41715](https://github.com/golang/go/issues/55949) and bug fixes to improve robustness. This release note describes what’s different between Istio 1.13.8 and Istio 1.13.9.

Reviewer: ```suggestion
This release contains a patch for [CVE-2022-41715](https://github.com/golang/go/issues/55949) and bug fixes to improve robustness. This release note describes what is different between Istio 1.13.8 and Istio 1.13.9.
```

Author: Done.

{{< relnote >}}

## Security update

- Patch for [CVE-2022-41715](https://github.com/golang/go/issues/55949). Replaces all uses of `stdlib` `regexp` with the Go 1.19.2 `stdlib` implementation. This will guard against DOS via malformed regular expressions.

## Changes

> - **Fixed** an issue where user can not delete the Istio Operator resource with revision if istiod is not running.  ([Issue #40796](https://github.com/istio/istio/issues/40796))

Reviewer: ```suggestion
- **Fixed** an issue where the user can not delete the Istio Operator resource with revision if istiod is not running.  ([Issue #40796](https://github.com/istio/istio/issues/40796))
```

Author: Done.
